src/main/java/org/commonjava/service/promote/client/content/ContentService.java

```diff
@@ -5,17 +5,19 @@
 import javax.ws.rs.*;
 import javax.ws.rs.core.Response;
 
-@Path("/api/content/{package}/{type: (hosted|group|remote)}/{name}/{path: (.*)}")
+@Path("/api/content")
 @RegisterRestClient(configKey="content-service-api")
 public interface ContentService
 {
     @GET
+    @Path("{package}/{type: (hosted|group|remote)}/{name}/{path: (.*)}")
```

Comment:
Does it need to add "/" at the beginning here ?

Reply:
I remember both ways work. Let's test it.